app/templates/_snippets.html

```diff
@@ -34,7 +34,7 @@
     <div class="col-md-8">
       <img src="{{dataset.image or 'https://assets.okfn.org/p/data/img/icon-128.png'}}"
         alt="{{dataset.title}}" class="logo" />
-      <div class="description">{{dataset.readme|truncate}} <a href="#readme">Read more</a></div>
+      <div class="description">{{dataset.readme|truncate|markdown}} <a href="#readme">Read more</a></div>
```

Comment:
Are you sure this is the right way round? You want this to be plain text with no markdown formatting i think -- and what happens if you truncate in the middle of markdown formatting ...